homeassistant/components/mastodon/notify.py

```diff
@@ -60,8 +70,59 @@ def __init__(self, api: Mastodon) -> None:
         self._api = api
 
     def send_message(self, message: str = "", **kwargs: Any) -> None:
-        """Send a message to a user."""
+        """Toot a message, with media perhaps."""
+        data = kwargs.get(ATTR_DATA)
+
+        media = None
+        mediadata = None
+        target = None
+        sensitive = False
+        content_warning = None
+
+        if data:
+            media = data.get(ATTR_MEDIA)
+            if media:
+                if not self.hass.config.is_allowed_path(media):
+                    LOGGER.warning(f"{media} is not a whitelisted directory")
```

Comment:
Don't use f-strings in log warnings, do like this instead
```suggestion
                    LOGGER.warning(f"'%s' is not a whitelisted directory", media)
```